CANDIDATE_INFORMATION.md

```diff
@@ -21,9 +21,10 @@ Google Meet is supported by the major browsers. Full requirements can be found [
 A link will be included on your interview invitation, click it to join.
 Once joined, your interviewer will ask you to share your screen.
 
-Please note, enabling your camera is optional.
-
-We do not assess your IDE setup.
+> **Note**
+> - Enabling your camera is optional.
+> - We do not assess your IDE setup.
+> - We currently forbid the use of GitHub Copilot and similar tools. They make it difficult to perform a meaningful assessment.
```

Comment:
Suggest removing the reasoning for forbidding Github Copilot. Most candidates would understand it, or ask about it. We could provide this statement as an guideline explanation in the [PROCESS_REMOTE.md](https://github.com/guardian/coding-exercises/blob/main/PROCESS_REMOTE.md)

```suggestion
> - Turning on your camera is optional.
> - We do not assess your IDE setup.
> - We currently do not allow the use of GitHub Copilot and similar tools.
```